src/Routing/Adapter/Lumen.php

```diff
@@ -371,4 +375,17 @@ public function gatherRouteMiddlewares($route)
         // Route middleware in Lumen is not terminated.
         return [];
     }
+
+    /**
+     * @param array  $route
+     * @param string $method
+     *
+     * @return array
+     */
+    private function setRouteMethods($route, $method)
+    {
+        return isset($route['methods'])
+            ? array_push($route['methods'], $method)
```

Comment:
What function/method I should use instead? array_push is not an alias, it's the native php function